docs/src/pages/components/tabs/react.mdx

```diff
@@ -353,56 +165,59 @@ In the `TabItem` component, use the `isDisabled` prop to make a Tab not clickabl
 
 <ComponentStyleDisplay componentName="Tabs" />
 
-### Global styling
+### Theme
 
-To override styling on all `Tabs` or `TabItems`, you can set the Amplify CSS variables, use the built-in `.amplify-tabs` or `.amplify-tabs-items`classes, or use pseudo-classes.
+You can customize the appearance of Tabs in your application using a [Theme](/theming).
 
-Note that children of `TabItem` must be responsible for their own styling.
+<Example>
+  <ThemeExample />
+  <ExampleCode>
+    ```jsx file=./examples/ThemeExample.tsx
 
-```css
-/* styles.css */
-:root {
-  /* background for the whole tab bar */
-  --amplify-components-tabs-background-color: var(
-    --amplify-colors-background-secondary
-  );
-  /* background for each tab */
-  --amplify-components-tabs-item-background-color: var(
-    --amplify-colors-background-primary
-  );
-}
-/* OR */
-.amplify-tabs {
-  background-color: var(--amplify-colors-background-secondary);
-}
+    ```
 
-.amplify-tabs-item {
-  background-color: var(--amplify-colors-background-primary);
-}
-```
+</ExampleCode>
+</Example>
 
-```jsx
-import './styles.css';
+### Global styling
 
-<Tabs>
-  <TabItem title="Tab 1">Content of Tab 1</TabItem>
-  <TabItem title="Tab 2">Content of Tab 2</TabItem>
-</Tabs>;
-```
+To override styling on all `Tabs` or `TabItems`, you can set the Amplify CSS variables, use the built-in `.amplify-tabs` or `.amplify-tabs-items`classes, or use pseudo-classes.
+
+Note that children of `TabItem` must be responsible for their own styling.
 
 <Example className="global-tabs-example">
-  <Tabs>
-    <TabItem title="Tab 1">
-      <DemoTabPanel>Content of Tab 1</DemoTabPanel>
-    </TabItem>
-    <TabItem title="Tab 2">
-      <DemoTabPanel>Content of Tab 2</DemoTabPanel>
-    </TabItem>
-  </Tabs>
+  <BasicTabs />
+  <ExampleCode>
+    ```css
+    /* styles.css */
+    :root {
```

Comment:
I think we're changing this to `[data-amplify-theme]` in instead of `:root`. Is that right, @reesscot?

Reply:
That sounds familiar; I updated this example to use [data-amplify-theme]

Reply:
One caveat to that is that customers don't _have_ to use the AmplifyProvider unless they are using a custom theme object. If we update our examples to always target `[data-amplify-theme]` then they will always have to use an AmplifyProvider, which doesn't seem ideal.